docs/source/roles/system_manager/migrate_an_shm.md

```diff
@@ -135,11 +135,11 @@ Run the following `Powershell` commands
 $userOuPath = (Get-ADObject -Filter * | Where-Object { $_.Name -eq "Safe Haven Research Users" }).DistinguishedName
 $users = Get-ADUser -Filter * -SearchBase "$userOuPath" -Properties *
 
-# Connect to AzureAD
-# Use the credentials for an AzureAD global admin (eg. `aad.admin.firstname.surname@<SHM domain>`)
+# Connect to Microsoft Entra ID
+# Use the credentials for a Microsoft Entra global admin (eg. `aad.admin.firstname.surname@<SHM domain>`)
```

Comment:
```suggestion
# Use the credentials for a Microsoft Entra ID global admin (eg. `aad.admin.firstname.surname@<SHM domain>`)
```

Reply:
Again, official guidance suggests no use of ID here